### DISCUSSION TITLE
Any thoughts on what labels might be appropriate for Geb?

### DISCUSSION THREAD

**Discussion by paulk-asert** [apache/groovy-geb#206] [General]
GitHub supports labels, e.g. for Groovy (the blue rounded rectangles):
![image](https://github.com/user-attachments/assets/de168766-1312-4889-9147-ffea7cf13a86)

What should we have for Geb?

**Comment by jonnybot0** [comment]
Obviously `groovy` and `apache`. `selenium`, `webdriver`, and `java` also seem sensible. 

There are a host of testing-related topics that make sense to add onto the repository as well:

- [functional-testing](https://github.com/topics/functional-testing)
- [browser-testing](https://github.com/topics/browser-testing)
- [end-to-end-testing](https://github.com/topics/end-to-end-testing)
- [e2e-testing](https://github.com/topics/e2e-testing)
- [test-automation](https://github.com/topics/test-automation)
- [test](https://github.com/topics/test)
- [testing-framework](https://github.com/topics/testing-framework)
- [testing-library](https://github.com/topics/testing-library)
- [bdd-style-testing-framework](https://github.com/topics/bdd-style-testing-framework)

**Comment by paulk-asert** [comment]
I added those suggestions and will close this but we can change at any time if folks finder better labels/topics.
